docs/src/main/paradox/scala/http/introduction.md

```diff
@@ -41,11 +60,22 @@ solely on the low-level API; make sure the Scala version is a recent release of
 
 Alternatively, you can bootstrap a new sbt project with Akka HTTP already
 configured using the [Giter8](http://www.foundweekends.org/giter8/) template:
+
+@@@ div { .group-scala }
 ```sh
 sbt -Dsbt.version=0.13.15 new https://github.com/akka/akka-http-scala-seed.g8
 ```
-More instructions can be found on the [template
-project](https://github.com/akka/akka-http-scala-seed.g8). Note, requires
+@@@
+@@@ div { .group-java }
+```sh
+sbt -Dsbt.version=0.13.15 new https://github.com/akka/akka-http-java-seed.g8
+```
+From there on the prepared project can be built using Gradle or Maven.
+@@@
+
+More instructions can be found on the @scala[[template
+project](https://github.com/akka/akka-http-scala-seed.g8)]@java[[template
+project](https://github.com/akka/akka-http-java-seed.g8)]. Note, requires
```

Comment:
This will probably have to be tweaked when the new supercharged Scala seed is done.